specs/shell/launch-clipboard-buffer-url-list.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: Launch tabs with clipboard URLs
+command: "pbpaste | xargs -n1 -I{} open {}"
+tags:
+  - pbpaste
+  - xargs
+  - macosx
+  - clipboard
+  - urls
```

Comment:
nit: we recommend fewer tags (Tags are visible in Warp in the workflows menu, so the more tags we have the more cluttered the UI is).

Thoughts on just including `pbpaste` here?

Reply:
no problem. pbpaste it is!